pom.xml

```diff
@@ -85,9 +85,33 @@
 
         <!-- Which Maven Central infra should be used -->
         <release.endpoint>https://s01.oss.sonatype.org/</release.endpoint>
+
+        <!-- Scala version used to build Kafka -->
+        <version.kafka.scala>2.12</version.kafka.scala>
+
+        <!-- ANTLR -->
+        <antlr.version>4.7.2</antlr.version>
+
+        <!-- Quarkus -->
+        <quarkus.version>1.12.0.Final</quarkus.version>
+
+        <!-- Databases -->
+        <version.postgresql.driver>42.2.14</version.postgresql.driver>
+        <version.mysql.server>5.7</version.mysql.server>
+        <version.mysql.driver>8.0.21</version.mysql.driver>
+        <version.mysql.binlog>0.25.0</version.mysql.binlog>
+        <version.mongo.server>3.6</version.mongo.server>
+        <version.mongo.driver>4.2.1</version.mongo.driver>
+        <version.sqlserver.driver>7.2.2.jre8</version.sqlserver.driver>
+        <version.oracle.driver>12.2.0.1</version.oracle.driver>
+        <version.db2.driver>11.5.0.0</version.db2.driver>
+        <version.cassandra.driver>3.5.0</version.cassandra.driver>
+        <version.cassandra>3.11.10</version.cassandra>
```

Comment:
Why are these not in the BOM?

Reply:
@gunnarmorling

`version.kafka.scala` is used as substring in artifact id
Quarkus and antlr are used also as plugin versions

Databases section mainly for server versions as these are used as tags in Docker images. 
I had two options, this one or having server versions defined here and drivers in BOM. This one looked better to me.

Reply:
> Quarkus and antlr are used also as plugin versions

Ah, and if you have `<pluginManagement>` in the BOM, it doesn't do anything?

> I had two options, this one or having server versions defined here and drivers in BOM. This one looked better to me.

Got it; so I think I'd still prefer to have as much as possible (i.e. the drivers) in the BOM.

Reply:
No, I checked BOMs of other projects and it seems that it is not usual to include in the bom plugins needed for build, only those that are used by the end user consuming the BOM.

For the latter, ok, I'll just keep comments on other sides about the need to keep it in sync